test/mongorito.test.js

```diff
@@ -1120,7 +1109,45 @@ describe ('Mongorito', function () {
     secondaryDb.close();
   });
 
+  it ('be able to yield a close call (issue #23)', function * (){
+    let secondaryDb = yield Mongorito.connect('localhost/mongorito_test_2');
+    yield secondaryDb.close();
```

Comment:
It throws an exception if I don't yield it now, when I try to reopen the same connect right away which is why it does bring value. This is the test case presented in issue #23 and it does seem to be a valid test case for close.